components/tutorial/tutorial_tip/tutorial_tip.tsx

```diff
@@ -144,45 +161,45 @@ export default class TutorialTip extends React.PureComponent<Props, State> {
         }
     }
 
-    public handleNext = (): void => {
-        if (this.state.currentScreen < this.props.screens.length - 1) {
-            this.setState({currentScreen: this.state.currentScreen + 1});
-            return;
-        }
-
-        if (this.props.telemetryTag) {
-            let tag = this.props.telemetryTag;
-
-            if (this.props.screens.length > 1) {
-                tag += '_' + (this.state.currentScreen + 1).toString();
-            }
-
-            if (this.state.currentScreen === this.props.screens.length - 1) {
-                tag += '_okay';
-            } else {
-                tag += '_next';
-            }
-
-            trackEvent('tutorial', tag);
-        }
-
-        const {currentUserId, actions} = this.props;
+    private handleSavePreferences = (autoTour: boolean, isNextStep?: boolean, isPrevStep?: boolean, jumpToStep?: number): void => {
```

Comment:
Could we change the way we pass in info about what the next step should be? I think we could do something like 
```typescript
private handleSavePreferences = (autoTour: boolean, nextStep: boolean | number): void => {
// ...
let stepValue = this.props.currentStep;
if (nextStep === true) {
  stepValue = stepVale + 1;
} else if (nextStep === false) {
  stepValue = stepValue - 1;
} else {
  stepValue = nextStep;
}
```

That would make it harder to pass unintentional combinations like both next and previous step is true, or including one of the next or  previous steps as true and passing a number when that was not intended.

Reply:
Thanks for the suggestion will change this.